src/dataconnect/build.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as prompt from "../prompt";
+import { handleBuildErrors } from "./build";
+import { GraphqlError } from "./types";
+
+describe("handleBuildErrors", () => {
+  let promptOnceStub: sinon.SinonStub;
+  before(() => {
+    sinon.verifyAndRestore();
+  });
```

Comment:
@joehan not sure why, but this test fails with `Attempted to wrap promptOnce which is already wrapped` without this clause here, even though the same thing is called in `afterEach`.